apps/web/modules/signup-view.tsx

```diff
@@ -619,57 +619,51 @@ export default function Signup({
               </div>
             </div>
           </div>
-          <div className="border-subtle lg:bg-subtle mx-auto mt-24 w-full max-w-2xl flex-col justify-between rounded-l-2xl pl-4 dark:bg-none lg:mt-0 lg:flex lg:max-w-full lg:border lg:py-12 lg:pl-12">
-            {IS_CALCOM && (
```

Comment:
remove

Reply:
Are we still wanting to show this even if a user is self hosted?

Reply:
I think we need to still have this IS_CALCOM check for social proof 

CC: @PeerRich

Reply:
Yeah we need to keep this for self-hosters